src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,32 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const commentsList = post.comments;
+  const isNotEmptyArray = commentsList.length > 0;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {'Posted by '}
+
+          <UserInfo user={post.user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">{post.body}</p>
+
+      {isNotEmptyArray ? (
+        <CommentList comments={commentsList} />
+      ) : (
+
+      // <hr />
```

Comment:
Why is it commented?

Reply:
> Why is it commented?

it was after 6-hour debugging of 3 issues,
what appeared after mine decision, what was separating ALL calculations (Roman Mirzoian said that it's smth 'business logic of app') 
on separated components files =)))

after QnA that issue was resolved,
but `<hr />` was commented cause it was one of ideas to resolve the issues with testing ))